sync_calendar.gs

```diff
@@ -117,7 +98,7 @@ function cleanCalender() {
   var sheet = SpreadsheetApp.getActive().getSheetByName('貼文日曆');
   let lastRow = sheet.getLastRow();
 
-  var range = sheet.getRange('D4:J'+lastRow);
+  var range = sheet.getRange('D27:J'+lastRow);
```

Comment:
other modifies are because of the inconsistences between github and AppScripts 😅

Reply:
This is just to reduce changes to the completed calendar entries.

Reply:
got it~